src/data/reducers/connectionStatus.js

```diff
@@ -0,0 +1,10 @@
+import { PING_RESPONSE } from '../actions/pingStudio';
+
+export default function connectionStatus(state = null, action) {
+  switch (action.type) {
+    case PING_RESPONSE:
```

Comment:
`connectionStatus` (and all other reducers in the application) is invoked every time an action creator passes data through Redux via an action -- essentially, every time a message hops on the bus. This allows it to pull in whatever action data it cares about, digest it, and return it. 

This is all very well and good, but the reducer probably doesn't care about every single action passing through -- it really only cares about the connection status. How can it pull out only what it needs? Well, remember `PING_RESPONSE` from our `pingStudio` action creator? This reducer listens for actions of that `type` so it can pull out their data and digest it. It's Redux convention to use a switch statement to accomplish this. If we wanted to, we could add other action type slugs to listen for, but we're just keeping it simple for now.